2.2.0/CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+# Changelog
+
+Astronomer Certified 2.2.0-1, 2021-08-24
+----------------------------------------
+
+User-facing CHANGELOG for AC 2.2.0+astro.1 from Airflow 2.2.0:
+
+### Bugfixes
+
+- [astro] Handle Istio containers with Kubernetes Executor Pod adoption (#1318) ([commit](https://github.com/astronomer/airflow/commit/26d52248b))
+- [astro] [AIRFLOW-5448] Handle istio-proxy for Kubernetes Pods (#62) ([commit](https://github.com/astronomer/airflow/commit/dc8b2a999))
```

Comment:
With these now being squashed I wasn't sure how we wanted to represent these, so I left them as-is.

Reply:
We can squash them here too, general idea is users can look here and visit actual commits in the release branch (v2-2-0 in this case)